c/ldv-memsafety-bitfields/test-bitfields-2.1_false-valid-free.c

```diff
@@ -7,8 +7,10 @@ struct A {
 	unsigned char a;
 	unsigned char b:2;
 	unsigned char c:2;
+	unsigned char pad1[2];
```

Comment:
This padding is required to avoid partial copy of 2 bytes of unsigned int d with added packed attribute